xbmc/pvr/addons/PVRClients.cpp

```diff
@@ -258,6 +258,40 @@ bool CPVRClients::GetClientName(int iClientId, std::string &strName) const
   return bReturn;
 }
 
+std::vector<SBackend> CPVRClients::GetBackendProperties() const
+{
+  std::vector<SBackend> backendProperties;
+  CSingleLock lock(m_critSection);
+
+  for (const auto &entry : m_clientMap)
+  {
+    const auto &client = entry.second;
+    
+    if (!client->ReadyToUse())
+      continue;
+
+    SBackend properties;
```

Comment:
and furthermore, using not using `auto` where it's not actually replacing something like `std::map<foo, struct bar>::const_iterator` would make things less confusing for readers too (comment doesn't apply to this instance, but some people sure like using the `auto` keyword as much as possible ;-))